cnd/src/swap_protocols/facade.rs

```diff
@@ -23,7 +23,9 @@ pub struct HanEtherereumHalightBitcoinCreateSwapParams {
     #[digest(prefix = "2001")]
     pub ethereum_absolute_expiry: Timestamp,
     #[digest(prefix = "2002")]
-    pub ethereum_amount: asset::Ether,
+    pub ethereum_amount: asset::Erc20Quantity,
+    #[digest(ignore)]
+    pub token_contract: EthereumIdentity,
```

Comment:
This should be part of the digest!
Two swaps on different ERC20 contracts are not the same :)

Reply:
Thanks for pointing it out! had a feeling it should be - this is a great reminder.
Will record it as follow up!